api/envoy/api/v2/route/route.proto

```diff
@@ -1062,6 +1065,13 @@ message DirectResponseAction {
   core.DataSource body = 2;
 }
 
+message Noop {
+  // If set to true, then when this route is hit, stream info (and hence access logs)
```

Comment:
From the user perspective, it's not clear what this actually does. Stream info as an internal concept. Is this feature generally usable without custom extensions?